### PR TITLE
Patching auth ticket to be python3 compatible

### PR DESCRIPTION
**Pull request by TilmanSchaefer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # - git push
 # - python setup.py sdist bdist_wheel upload --sign
 
-__version__ = '3.4.2'
+__version__ = '3.4.2.1'
```
Comment (cdent) [inline setup.py:15]: Don't change this, I'll take care of it when I package a release.

Also, when I do change it the new version number will be `3.4.3`, following semver.

**Comment by cdent** [comment]
This looks good, but please remove the version number change (see other comment).

Also, out of curiosity: Are you using this code and this is broken for you, or was it something you stumbled upon?

Thanks.

**Comment by TilmanSchaefer** [comment]
Thanks for the feedback. Yes we are using this code and it broke during a python version upgrade. I don't know if there are other parts in this module that would need fixing but at least it works for us now.

**Comment by cdent** [comment]
Released as 3.4.3 : https://pypi.org/project/Paste/

Thanks very much for the patch.

**Pull request merged**